simapp/go.mod

```diff
@@ -19,8 +19,8 @@ require (
 	github.com/cometbft/cometbft v0.37.1-0.20230411132551-3a91d155e664
 	github.com/cosmos/cosmos-db v1.0.0-rc.1
 	// this version is not used as it is always replaced by the latest Cosmos SDK version
-	github.com/cosmos/cosmos-sdk v0.48.0
-	github.com/cosmos/gogoproto v1.4.7
+	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20230418140744-0dde947d0ab7
```

Comment:
simd version --long is looks nicer if we revert this one.

Reply:
Oh I see, because these two files have the replace directive. I'll push a change correcting these two.